extension/pprofextension/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.58.0
 	github.com/stretchr/testify v1.8.0
 	go.opentelemetry.io/collector v0.58.0
-	go.uber.org/atomic v1.9.0
+	go.uber.org/atomic v1.10.0
```

Comment:
Looks like this bump is causing lint to fail due to a deprecation

Reply:
Fixed.